spot-client/src/spot-remote/ui/utils/formatPhoneNumber.js

```diff
@@ -0,0 +1,11 @@
+import { parsePhoneNumberFromString } from 'libphonenumber-js/max';
+
+/**
+ * Pretty prints given phone number. Will return a formatted text if a valid phone number string is given.
+ *
+ * @param {string} [phoneNumber] - A not pretty phone number string, for example: '+12343334444'.
+ * @returns {?string}
+ */
+export function formatPhoneNumber(phoneNumber) {
+    return (phoneNumber && parsePhoneNumberFromString(phoneNumber))?.formatInternational();
```

Comment:
What would happen if undefined or an empty string is passed into parsePhoneNumberFromString? That could remove some of the defensive checking.

Reply:
it throws that's why it needs to be there